examples/film.py

```diff
@@ -130,17 +130,17 @@ def main(argv=None):
             rating = None
             while not rating or (rating > 5 or rating <= 0):
                 try:
-                    rating = int(input("Rating (on five): "))
+                    rating = int(eval(input("Rating (on five): ")))
```

Comment:
shouldn't `int(input())` be enough? Why do we need `eval()?` Also in the other cases, isn't a string fine?